app/src/main/java/com/duckduckgo/app/survey/api/SurveyService.kt

```diff
@@ -24,7 +24,7 @@ import retrofit2.http.GET
 @ContributesServiceApi(AppScope::class)
 interface SurveyService {
 
-    @GET("https://staticcdn.duckduckgo.com/survey/v2/survey-mobile.json")
+    @GET("https://ddg-sandbox.s3.amazonaws.com/survey/v2/survey-mobile.json")
```

Comment:
Unrelated change. Seems a test endpoint for surveys.

Reply:
good catch, removing